src/GeographicalFeatures/CriticalFacilities/Form/index.js

```diff
@@ -228,23 +233,51 @@ const FeatureForm = ({ feature, isEditForm, posting, onCancel }) => {
       )}
       {/* end: longitude & latitude */}
 
-      {/* start: custodians */}
-      <Form.Item
-        label="Custodians"
-        title="Responsible Agencies e.g Police Force"
-        name={['relations', 'custodians']}
-      >
-        <SearchableSelectInput
-          onSearch={(optns = {}) => {
-            return getAgencies(optns);
-          }}
-          optionLabel={(custodian) => get(custodian, 'name')}
-          optionValue="_id"
-          initialValue={get(feature, 'relations.custodians', [])}
-          mode="multiple"
-        />
-      </Form.Item>
-      {/* end: custodians */}
+      {/* start: custodians & ownership */}
+      <Row justify="space-between">
+        {/* start:custodians */}
+        <Col xs={24} sm={24} md={11}>
+          <Form.Item
+            label="Custodians"
+            title="Responsible Agencies e.g Police Force"
+            name={['relations', 'custodians']}
+          >
+            <SearchableSelectInput
+              onSearch={(optns = {}) => {
+                return getAgencies(optns);
+              }}
+              optionLabel={(custodian) => get(custodian, 'name')}
+              optionValue="_id"
+              initialValue={get(feature, 'relations.custodians', [])}
+              mode="multiple"
+            />
+          </Form.Item>
+        </Col>
+        {/* end:custodians */}
+
+        {/* start:ownership */}
+        <Col xs={24} sm={24} md={11}>
+          <Form.Item
+            label="Ownership"
+            name={['relations', 'ownership', '_id']}
```

Comment:
Hello @Mloweedgar, Thanks for this PR, I think there is no need or _id here for it to work if I'm not mistaken.

Reply:
We just be `name={['relations','ownership']}` only